cpp/src/aztec/transcript/transcript.cpp

```diff
@@ -150,6 +164,10 @@ void Transcript::apply_fiat_shamir(const std::string& challenge_name /*, const b
     // TODO(Cody): Coupling: this line insists that the challenges in the manifest
     // are encountered in the order that matches the order of the proof construction functions.
     // Future architecture should specify this data in a single place (?).
+    // info("apply_fiat_shamir: challenge name match");
```

Comment:
Leaving a lot of logging statements in comments. Intended? OK if so.

Reply:
Yeah I wasn't planning on leaving them in the final PR but I'd actually like to while were in major debugging mode. I wind up putting these statements in every branch for debugging anyway.